nncf/common/pruning/operations.py

```diff
@@ -199,7 +199,10 @@ def generate_output_mask(cls, node: NNCFNode, graph: NNCFGraph,
         :return: Filled input masks.
         """
         input_edges = graph.get_input_edges(node)
-        input_masks = get_input_masks(node, graph)
+        previous_nodes = [edge.from_node for edge in input_edges]
+        input_masks = [input_node.data['output_mask'] for input_node in previous_nodes]
+        input_masks = [input_mask[node.node_name] if isinstance(input_mask, dict) else input_mask
+                       for input_mask in input_masks]
```

Comment:
Please use `get_input_masks`

Reply:
I reverted the use of `get_input_masks` in here. When I use `get_input_masks` in concat, tests/torch/nas/test_all_elasticity.py is failed. And I found this is due to the order of the result of `graph.get_previous_nodes(node)` and the above one is different. I think this is the reason that the current code in develop also not use `get_input_masks`. https://github.com/openvinotoolkit/nncf/blob/7b1b3b430cdc03e592aacc125282910de97297ee/nncf/common/pruning/operations.py#L199-L201